rocAL/rocAL/source/bounding_box_graph.cpp

```diff
@@ -110,45 +115,46 @@ void BoundingBoxGraph::update_random_bbox_meta_data(CropCordBatch *_random_bbox_
         BoundingBoxCords bb_coords;
         BoundingBoxLabels bb_labels;
         BoundingBoxCord crop_box;
-        crop_box.x = crop_cords[i]->crop_x;
-        crop_box.y = crop_cords[i]->crop_y;
-        crop_box.w = crop_cords[i]->crop_width;
-        crop_box.h = crop_cords[i]->crop_height;
+        crop_box.l = crop_cords[i]->crop_left;
+        crop_box.t = crop_cords[i]->crop_top;
+        crop_box.r = crop_cords[i]->crop_right;
+        crop_box.b = crop_cords[i]->crop_bottom;
+    //    std::cout<<"CROP Co-ordinates in bounding box grpah: lxtxrxb::\t"<<crop_box.l<<"x\t"<<crop_box.t<<"x\t"<<crop_box.r<<"x\t"<<crop_box.b<<"x\t"<<std::endl;
+
         for (uint j = 0; j < bb_count; j++)
         {
             int m = j * 4; // change if required
 
             //Mask Criteria
-            auto left = int(crop_box.x);
```

Comment:
Why round here? It is better to keep coordinates in float for all computations to avoid rounding error

Reply:
This change has been already done .